hpat/datatypes/hpat_pandas_series_functions.py

```
@@ -256,6 +256,117 @@ def hpat_pandas_series_values_impl(self):
+
+    Parameters
+    -----------
+    normalize: :obj: boolean, default False
+        If True then the object returned will contain the relative frequencies of the unique values
+    sort: :obj: boolean, default True
+        Sort by frequencies
+    ascending: :obj: boolean, default False
+        Sort in ascending order
+    bins: :obj: integer, default None
+        *unsupported*
+    dropna: :obj: boolean, default True
+        Skip counts of NaN
+        *unsupported* for String
+
+    Returns
+    -------
+    :obj:`pandas.Series`
+    """
+
+    _func_name = 'Method value_counts().'
+
+    if not isinstance(self, SeriesType):
+        raise TypingError('{} The object must be a pandas.series. Given: {}'.format(_func_name, self))
+
+    if not isinstance(normalize, types.Omitted) and normalize:
+        raise TypingError('{} Unsupported parameters. Given normalize: {}'.format(_func_name, normalize))
+
+    if not isinstance(sort, (types.Omitted, types.Boolean, bool)):
+        raise TypingError('{} The object must be boolean. Given sort: {}'.format(_func_name, sort))
+
+    if not isinstance(ascending, (types.Omitted, types.Boolean, bool)):
+        raise TypingError('{} The object must be boolean. Given ascending: {}'.format(_func_name, ascending))
+
+    if not isinstance(bins, (types.Omitted, types.NoneType)) and bins is not None:
+        raise TypingError('{} Unsupported parameters. Given bins: {}'.format(_func_name, bins))
+
+    if not isinstance(dropna, (types.Omitted, types.Boolean, bool)):
+        raise TypingError('{} The object must be boolean. Given dropna:{}'.format(_func_name, dropna))
+
+    if isinstance(self.data, StringArrayType):
+        def hpat_pandas_series_value_counts_str_impl(self, normalize=False, sort=True, ascending=False, bins=None, dropna=True):
+            if dropna:
+            # TODO: add nan handling
+                print("Not supported yet")
```

I think better to raise exception instead of printing, e.g. `ValueError`.